### PR TITLE
DOC-9894 PR #120236 - pkg/sql/sqlstats: move failures out of stmt key/meta, into stats (1 of 2)

### PR DESCRIPTION
**Pull request by florence-crl**
Fixes DOC-9894

In statement-details.md include file, replaced Failed with Failure Count. Updated images.

rendered preview: [Statements page](https://deploy-preview-18450--cockroachdb-docs.netlify.app/docs/v24.1/ui-statements-page.html)

### CODE REVIEW

File: src/current/images/v24.1/statement-fingerprint.png
Comment (abarganier) [inline src/current/images/v24.1/statement-fingerprint.png:1]: Is there any difference in the columns between the old and new with this image? Just curious as I wasn't able to notice one myself.

**Review by abarganier — APPROVED** [review]
LGTM

File: src/current/_includes/v24.1/ui/statement-details.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The **Overview** section displays the SQL statement fingerprint and execution at
 **Database** | The database on which the statements executed.
 **Application Name** | The name specified by the [`application_name`]({{ link_prefix }}show-vars.html#supported-variables) session setting. Click the name to view all statements run by that application.
 **Fingerprint ID** | The ID of the statement fingerprint in hexadecimal format. It may be used to query the [`crdb_internal.statement_statistics`]({{ link_prefix }}crdb-internal.html#fingerprint_id-column) table.
-**Failed?** | Whether the statement failed to execute.
+**Failure Count** | The total number of executions of this statement fingerprint that failed.
```
Comment (taroface) [inline src/current/_includes/v24.1/ui/statement-details.md:18]: Optional suggestion

```suggestion
**Failure Count** | The total number of failed executions of this statement fingerprint.
```

**Review by taroface — APPROVED** [review]
LGTM! One optional suggestion

File: src/current/_includes/v24.1/ui/statement-details.md (hunk 1 above)
Comment (florence-crl) [reply src/current/_includes/v24.1/ui/statement-details.md]: updated

File: src/current/images/v24.1/statement-fingerprint.png
Comment (florence-crl) [reply src/current/images/v24.1/statement-fingerprint.png:1]: Yes, **SQL CPU Time** header in new image vs **CPU Time" header in old image. Also the row values match the values in the Details image.

**Review by florence-crl — COMMENTED** [review]
TFTRs

**Pull request merged**